packages/main/src/i18n/messagebundle.properties

```diff
@@ -25,6 +25,12 @@ BUTTON_ARIA_TYPE_REJECT=Negative Action
 #XACT: ARIA announcement for the emphasized button
 BUTTON_ARIA_TYPE_EMPHASIZED=Emphasized
 
+#XACT: Aria information for the Date Picker
```

Comment:
This is for the translation team, they don't really know what this means, so simply put a text such as "Date" IMO